### PR TITLE
chore: use gogofaster instead of gofast

### PR DESCRIPTION
**Pull request by NamanJain8**
Switch to gogofaster as it does not produce the extra XXX_ fields that increase up the space of protobuf.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1639)

### CODE REVIEW

File: pb/gen.sh [hunk 1]
```diff
@@ -5,4 +5,4 @@
 
 # You might need to go get -v github.com/gogo/protobuf/...
 
-protoc --gofast_out=plugins=grpc:. --gofast_opt=paths=source_relative -I=. badgerpb3.proto
+protoc --gogofaster_out=plugins=grpc:. --gogofaster_opt=paths=source_relative -I=. badgerpb3.proto
```
Comment (jarifibrahim) [inline pb/gen.sh:8]: Do we need this grpc plugin? Can we remove it?

**Review by jarifibrahim — APPROVED** [review]
Looks good to me. @NamanJain8 double-check this doesn't break anything. Also, check what these `XXX_...` fields were storing.

**Review by manishrjain — COMMENTED** [review]
Yeah. Just ensure it doesn't break anything. :lgtm: 

> *[Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1639#-:-MR_ebPA9c8-yjij4oG_:bnul538)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @NamanJain8)

File: pb/gen.sh (hunk 1 above)
Comment (NamanJain8) [reply pb/gen.sh]: No, we don't need. Yes, we can remove it. Removed.

**Review by jarifibrahim — APPROVED** [review]
Verify this with the older version of badger. This change shouldn't break anything.

**Comment by NamanJain8** [comment]
> Verify this with the older version of badger. This change shouldn't break anything.

Verified, the backup and restore work fine. Also, read-write can be performed seamlessly across this change.

**Pull request merged**